docs/usage/offline.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Offline Mode
+permalink: /usage/offline/
+parent: Usage
+nav_order: 4
+---
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+## Offline mode
+
+{: .note }
+Features and flags with the `experimental` prefix might change or be removed with only a minor version update.
+
+OSV-Scanner now supports offline scanning as an experimental feature. Offline scanning checks your project against a local database instead of calling the OSV.dev API.
```

Comment:
Couple of things I should are important to note:
  * where the databases are sourced from, for greater transparency (I imagine also folks dealing with air gapped environments will also appreciate being able to easily know what traffic to look for / allow in their general networks)
  * commits are not supported
  * local tools/libraries are not required aka version comparison is done using Go-based implementation of each ecosystems version spec, rather than trying to callout to dependency managers
  * people are encouraged to report if they think versions are being compared incorrectly (most of the ecosystems should be pretty robust, but some like Maven have a lot of grey areas and ultimately real-world data is going to be the best feedback)